exporter/containerimage/exptypes/types.go

```diff
@@ -11,7 +11,7 @@ const (
 	ExporterImageConfigDigestKey = "containerimage.config.digest"
 	ExporterImageDescriptorKey   = "containerimage.descriptor"
 	ExporterInlineCache          = "containerimage.inlinecache"
-	ExporterBuildInfo            = "containerimage.buildinfo"
+	ExporterBuildInfo            = "containerimage.buildinfo" // deprecated: Build information format has been deprecated and will be removed in a future release. Use provenance attestations instead.
```

Comment:
I _think_ the official format is to use a capital (`Deprecated`)


```suggestion
	ExporterBuildInfo            = "containerimage.buildinfo" // Deprecated: Build information format has been deprecated and will be removed in a future release. Use provenance attestations instead.
```